src/python_testing/TC_AVSM_2_13.py

```diff
@@ -0,0 +1,241 @@
+                "TH sends the VideoStreamAllocate command with the same arguments as step 8.",
+                "DUT responds with `VideoStreamAllocateResponse` command with a valid `VideoStreamID`",
+                "Verify that this VideoStreamID is the same as `aVideoStreamID`",
+            ),
+            TestStep(
+                11,
+                "TH reads AllocatedVideoStreams attribute from CameraAVStreamManagement Cluster on TH_SERVER",
+                "Verify the number of allocated video streams in the list is 1.",
+            ),
+        ]
+
+    @run_if_endpoint_matches(
+        has_feature(Clusters.CameraAvStreamManagement, Clusters.CameraAvStreamManagement.Bitmaps.Feature.kVideo)
+    )
+    async def test_TC_AVSM_2_13(self):
+        endpoint = self.get_endpoint(default=1)
+        cluster = Clusters.CameraAvStreamManagement
+        attr = Clusters.CameraAvStreamManagement.Attributes
+        commands = Clusters.CameraAvStreamManagement.Commands
+
+        self.step("precondition")
+        # Commission DUT - already done
+
+        self.step(1)
+        aFeatureMap = await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attr.FeatureMap)
+        logger.info(f"Rx'd FeatureMap: {aFeatureMap}")
+        vdoSupport = aFeatureMap & cluster.Bitmaps.Feature.kVideo
+        asserts.assert_equal(vdoSupport, cluster.Bitmaps.Feature.kVideo, "Video Feature is not supported.")
+
+        self.step(2)
+        aAllocatedVideoStreams = await self.read_single_attribute_check_success(
+            endpoint=endpoint, cluster=cluster, attribute=attr.AllocatedVideoStreams
+        )
+        logger.info(f"Rx'd AllocatedVideoStreams: {aAllocatedVideoStreams}")
+        asserts.assert_equal(len(aAllocatedVideoStreams), 0, "The number of allocated video streams in the list is not 0")
+
+        self.step(3)
+        aRankedStreamPriorities = await self.read_single_attribute_check_success(
+            endpoint=endpoint, cluster=cluster, attribute=attr.RankedVideoStreamPrioritiesList
+        )
+        logger.info(f"Rx'd RankedVideoStreamPrioritiesList: {aRankedStreamPriorities}")
+
+        self.step(4)
+        aRateDistortionTradeOffPoints = await self.read_single_attribute_check_success(
+            endpoint=endpoint, cluster=cluster, attribute=attr.RateDistortionTradeOffPoints
+        )
+        logger.info(f"Rx'd RateDistortionTradeOffPoints: {aRateDistortionTradeOffPoints}")
+
+        self.step(5)
+        aMinViewport = await self.read_single_attribute_check_success(
+            endpoint=endpoint, cluster=cluster, attribute=attr.MinViewport
+        )
+        logger.info(f"Rx'd MinViewport: {aMinViewport}")
+
+        self.step(6)
+        aVideoSensorParams = await self.read_single_attribute_check_success(
+            endpoint=endpoint, cluster=cluster, attribute=attr.VideoSensorParams
+        )
+        logger.info(f"Rx'd VideoSensorParams: {aVideoSensorParams}")
+
+        self.step(7)
```

Comment:
What is this step actually verifying?  The MaxEncodedPixelRate is never used nor checked anywhere in the test case

Reply:
The pixel rate of all the streams combined should actually be less than the camera's MaxEncodedPixelRate. For now, I have added a basic sanity check that the current stream allocation parameters being used does not exceed the MaxEncodedPixelRate.